src/PowerShellEditorServices/Services/CodeLens/ReferencesCodeLensProvider.cs

```diff
@@ -113,9 +113,18 @@ public CodeLens ResolveCodeLens(CodeLens codeLens, ScriptFile scriptFile)
                         continue;
                     }
 
+                    DocumentUri uri = DocumentUri.From(foundReference.FilePath);
+                    // For any vscode-notebook-cell, we need to ignore the backing file on disk.
+                    if (scriptFile.DocumentUri.Scheme == "vscode-notebook-cell" &&
+                        uri.Path == scriptFile.DocumentUri.Path &&
```

Comment:
Does this need to be case-insensitive?

Reply:
They _should_ be identical ... because in both cases it's using the actual file path on the file system.